back/app/services/user_display_name_service.rb

```diff
@@ -88,4 +97,10 @@ def admin?
   def initial(name)
     "#{name[0]}."
   end
+
+  # Generate first and last name in the format 'user' '123456'
+  def create_anonymous_names(user)
+    user.first_name = 'user'
```

Comment:
In `first_name` the value was "User", while here it's "user" (no capital first letter). Is this on purpose?